src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
+        if isinstance(batch, EnsembleSeries):
+            if batch.name == self._id_col:
+                batch = batch.rename("result")
+            batch = EnsembleFrame.from_dask_dataframe(batch.to_frame())
+            if len(on) > 1:
+                batch = batch.reset_index()
+                # Need to overwrite the meta manually as the multiindex will be
+                # interpretted by dask as a single "index" column
+                batch._meta = TapeFrame(columns=on + ["result"])
+
+                if by_band:
+                    batch = batch.categorize(self._band_col).pivot_table(
+                        index=on[0], columns=self._band_col, aggfunc="sum"
+                    )
+
+                    # Need to once again reestablish meta for the pivot
+                    band_labels = batch.columns.values
+                    out_cols = []
+                    for col in ["result"]:
```

Comment:
Why do we need this loop here?

Reply:
good catch, I think I left this open to further generalization but for the EnsembleSeries case it should always just be a single result, will remove this.

Reply:
resolved by latest commit

Reply:
Thank you!